client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -70,14 +76,17 @@ class EmailProvidersComparison extends React.Component {
 	};
 
 	goToAddGSuite = () => {
-		const { domain, currentRoute, selectedSiteSlug } = this.props;
+		const { domain, currentRoute, gSuiteProduct, selectedSiteSlug } = this.props;
```

Comment:
We should confirm we don't need to make related changes in the new email providers comparison page added in #51307. I think it's just logic cleanup rather than any blocking changes, though.